src/include/miopen/sequences.hpp

```diff
@@ -32,10 +32,10 @@
 #include <boost/range/algorithm/find.hpp>
 
 #include <algorithm>
+#include <array>
 #include <cassert>
 #include <tuple>
 #include <vector>
-#include <array>
```

Comment:
@DrizztDoUrden may I ask why this line is removed?

Reply:
I don't remember how it got two lines of `#include <array>`, but looks like at some point I have accidentally added a second one. Than I have removed the one which placed in wrong line (not alphabetically sorted). I can revert it to how it has been before if you want.

Reply:
Ah I see now. Thanks! I took a glance at it and thought it was removed. I recall adding similar edit due to Ubuntu 22.04 update, but it is not the same issue.

Reply:
Maybe I've got the second line from merge. Any way, it got sorted alphabetically by me. xD